custom-events/functions/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const {onCustomEventPublished} = require("firebase-functions/v2/eventarc");
+const logger = require("firebase-functions/logger");
+const admin = require("firebase-admin");
+
+admin.initializeApp();
+
+exports.onimageresized = onCustomEventPublished(
```

Comment:
Suggest tags for main part of sample, like:

// [START imageresizedEvent]

Probably not closing until after the custom logic, unless we want that separate.  Again, good to check with Jeff.

Reply:
Done.